apps/native-ui-storybook/components/Text/Variants.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Box, Center, Text, VStack } from '@utilitywarehouse/native-ui';
+
+const TextWrap = ({ title, children }: any) => (
```

Comment:
from my perspective, i can be 100% wrong 😅 ... we should encourage the use of Stack rather than having this component instead, why:
- because we are relying on the line heights to achieve the visual separations of the text elements
- one text is pass as a prop and the other as children 🤔 why not using two stacked `<Text />` with different font sizes and styles? 
- The wrapping of the children elements should  be controlled from the container/parent element when possible